spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/java/com/example/SecretManagerWebController.java

```diff
@@ -52,6 +49,11 @@
 	@Value("${my-application-secret}")
 	private String myApplicationSecretValue;
 
+	// Another way to access these properties is to @Autowire a @ConfigurationProperties-annotated class
```

Comment:
Clarify "these"?

Reply:
Done - reworded to say "another way to access these secrets..."